### ISSUE TITLE
relabel bug

### ISSUE THREAD

**Issue by sepandhaghighi** [sepandhaghighi/pycm#293]
#### Description
relabel method mapping order bug
#### Steps/Code to Reproduce
```pycon
>>> y_actu = [2, 0, 2, 2, 0, 1, 1, 2, 2, 0, 1, 2]
>>> y_pred = [0, 0, 2, 1, 0, 2, 1, 0, 2, 0, 2, 2]
>>> cm = ConfusionMatrix(y_actu, y_pred)
>>> cm.relabel({0:"L1",1:"L2",2:"L3"})
>>> cm.relabel({'L3': 6, 'L1': 3, 'L2': 3})
```
#### Expected Behavior
Relabeling
#### Actual Behavior
```
Traceback (most recent call last):
  File "<pyshell#12>", line 1, in <module>
    cm.relabel({'L3': 6, 'L1': 3, 'L2': 3})
  File "C:\Users\Sepkjaer\AppData\Local\Programs\Python\Python35-32\lib\site-packages\pycm-2.6-py3.5.egg\pycm\pycm_obj.py", line 633, in relabel
    raise pycmMatrixError(MAPPING_CLASS_NAME_ERROR)
pycm.pycm_error.pycmMatrixError: Mapping class names error
```
#### Operating System
Windows 10
#### Python Version
Python 3.5
#### PyCM Version (Use : `pycm.__version__`)
2.6

**Comment by elliestath** [comment]
Hey, I have the same error. I try to rename my class labels for the plot. I do:

```
cm.relabel(mapping={0:"L1",1:"L2",2:"L3"})
cm = ConfusionMatrix(actual_vector=y_actu, predict_vector=y_pred, is_imbalanced = True, digit = 3)
cm.plot(cmap=plt.cm.Blues,normalized=True, number_label=True, title = 'Confusion Matrix', plot_lib="matplotlib")
```

and I get 

`pycmMatrixError: Mapping class names error`

Any ideas on what I am doing wrong? Thanks!

**Comment by elliestath** [comment]
hey, thanks for the response!
I guess the problem was that I was defining 5 classes and had 6 labels.
Now, it seems to work.
However, do you have any solution to reduce the number of decimal points shown in the table? I tried to use the "digit" option but it doesn'e seem to work. Thank you!

**Comment by elliestath** [comment]
right, I need it for `cm.plot(cmap=plt.cm.Blues,normalized=True, number_label=True, title = 'Confusion Matrix', plot_lib="matplotlib")`

**Comment by sepandhaghighi** [comment]
Currently, we set the `plot` method decimal point to 5.
We will fix this issue (#430) in a future release, thanks for your report.

SH

**Comment by elliestath** [comment]
will be waiting, thanks!
